cli/setup.go

```diff
@@ -509,13 +509,13 @@ func (opts *setupOptionsType) getTenantToken(
 		return errors.Wrap(err,
 			"Reading tenant token FAILED.")
 	}
-	dataJson := make(map[string]string)
+	dataJson := make(map[string]interface{})
 	err = json.Unmarshal(data, &dataJson)
 	if err != nil {
 		return errors.Wrap(err,
 			"Error parsing JSON response.")
 	}
-	opts.tenantToken = dataJson["tenant_token"]
+	opts.tenantToken = dataJson["tenant_token"].(string)
```

Comment:
Question:
For future proofing this, perhaps we should check the typecast here, so that changing the format on the server, will not crash the client?

Reply:
You're right, this is too lazy. I updated the PR with an implicit type check in the Unmarshal call.